core/controllers/base.py

```diff
@@ -244,7 +247,7 @@ def dispatch(self):
                     '%s: payload %s',
                     e, self.payload)
 
-                self.handle_exception(e, self.app.debug)
+                return self.handle_exception(e, self.app.debug)
```

Comment:
Found out the problem. This return is necessary. We cannot remove it although `handle_exception` returns `None`.

Reply:
Yes, because execution continues going onward. An alternative is to do

    self.handle_exception(e, self.app.debug)
    return